sys/include/net/eui64.h

```diff
@@ -83,6 +83,80 @@ static inline void eui64_clear_group(eui64_t *addr)
     addr->uint8[0] &= ~EUI64_GROUP_FLAG;
 }
 
+/**
+ * @brief Board-specific function to supply a short address to a netdev
+ *
+ * @note  Implement this function in your board code if the board
+ *        provides the means to supply a unique address to a netdev.
+ *
+ * @warning Don't call this function directly, use @ref eui_short_get() instead.
+ *
+ * @param[in] netdev    The network device that requested the address
+ * @param[out] addr     The dedicated address for the netdev
+ *
+ */
+size_t board_get_eui_short(netdev_t *netdev, uint16_t *addr);
+
+/**
+ * @brief Board-specific function to supply an EUI-64 to a netdev
+ *
+ * @note  Implement this function in your board code if the board
+ *        provides the means to supply a unique address to a netdev.
+ *
+ * @warning Don't call this function directly, use @ref eui64_get() instead.
+ *
+ * @param[in] netdev    The network device that requested the address
+ * @param[out] addr     The dedicated address for the netdev
+ *
+ */
+size_t board_get_eui64(netdev_t *netdev, eui64_t *addr);
+
+/**
+ * @brief Generates an short address for the netdev interface.
+ *
+ * @note It is possible to supply a board-specific, constant addres
+ *       by implementing @ref board_get_eui_short.
+ *       If no such function is availiable, this will fall back to
+ *       @ref luid_get_short.
+ *
+ * @param[in] netdev    The network device for which the address is
+ *                      generated.
+ *                      Will be passed on to @ref board_get_eui48.
+ * @param[out] addr     The generated short address
+ *
+ */
+static inline void eui_short_get(netdev_t *netdev, uint16_t *addr)
```

Comment:
Why is this in `eui64.h`. Neither is the short address of IEEE 802.15.4 an EUI, nor has it anything to do with an EUI-64 (except that they are both used in IEEE 802.15.4 as addresses). This function and `board_get_eui_short` don't belong here. You've put the implementation for `board_get_…()` already in `l2util` (which given that `l2util` might not be compiled in in any case is somewhat of a bargain IMHO, but let's discuss this elsewhere), so why not put it there as `l2util_generate_short_addr()`?

Reply:
Then it should also be `l2util_generate_euiXX()` for consistency.

Reply:
Yes, i wrote the other comments before I realized, that the other functions are also misplaced.